src/OrleansRuntime/Cancellation/CancellationSourcesExtension.cs

```diff
@@ -13,16 +12,17 @@ namespace Orleans.Async
     internal class CancellationSourcesExtension : ICancellationSourcesExtension
     {
         private readonly static Lazy<TraceLogger> _logger = new Lazy<TraceLogger>(() =>
-            TraceLogger.GetLogger("CancellationSourcesExtension", TraceLogger.LoggerType.Runtime));
+            TraceLogger.GetLogger(typeof(CancellationSourcesExtension).Name, TraceLogger.LoggerType.Runtime));
```

Comment:
just so you know, we are now using c# 6, so these kind of things could simply be `nameof(CancellationSourcesExtension)`